data/src/config/away.rs

```diff
@@ -0,0 +1,21 @@
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Copy, Default, Deserialize)]
+pub struct Away {
```

Comment:
This should live in / under buffer since it's on buffer and not top level to Config